src/cpp/session/modules/SessionFind.cpp

```diff
@@ -583,7 +586,7 @@ class GrepOperation : public boost::enable_shared_from_this<GrepOperation>
          }
          catch (const std::ios_base::failure& e)
          {
-            error = Error(e.code().value(), e.what(), ERROR_LOCATION);
+            error = systemError(e.code().value(), ERROR_LOCATION);
```

Comment:
I think you should still be able to pass `e.what()` as the message here if it would be valuable.